rose/client/main.py

```diff
@@ -68,16 +70,40 @@ def send_message(self, msg):
         self.client.sendLine(str(msg))
 
 
+def load_driver_module(file_path):
+    """
+    Title Try to load the driver module from file
+
+    Arguments:
+      file_path (str): The path to the driver module
+
+    Returns:
+        Driver module (module)
+
+    Raises:
+        ImportError if the module cannot be loaded
+    """
+    module_dir, module_name = os.path.split(os.path.splitext(file_path)[0])
+    fp, pathname, description = imp.find_module(module_name, [module_dir])
+    try:
+        return imp.load_module(module_name, fp, pathname, description)
+    finally:
+        # Since we may exit via an exception, close fp explicitly.
+        if fp:
+            fp.close()
+
+
 def main():
     logging.basicConfig(level=logging.INFO)
     if len(sys.argv) < 2:
         log.info('usage: rose-client drive-module')
         sys.exit(2)
+    try:
+        driver_mod = load_driver_module(sys.argv[1])
+    except ImportError:
+        print "Couldn't load driver file"
```

Comment:
We want to include the path of the file and the text of the error. This will make it easier to debug the issue when a user is complaining about a failure - see the example in my comment in a previous version of this PR.